hs_composite_resource/tests/test_composite_resource_auto_aggregate.py

```diff
@@ -2,6 +2,7 @@
 
 from django.test import TransactionTestCase
 from django.contrib.auth.models import Group
+from django.core.files.uploadedfile import UploadedFile
 
 from hs_core.testing import MockIRODSTestCaseMixin
 from hs_core import hydroshare
```

Comment:
We should start standardizing. You can do a PyCharm "Optimize Imports" or equivalent, if you don't already do this as a regular practice. Not sure if you did already here, but since I'm new, just voicing this tip.

Reply:
Good tip. Normally, I use PyCharm to optimize imports.